README.md

```diff
@@ -38,14 +38,16 @@ export GOOGLE_CLOUD_PROJECT="<name of your project>" e.g. export GOOGLE_CLOUD_PR
 
 ## Running
 
-There are two ways of running this service
+There are several ways of running this service
```

Comment:
README needs updating to describe additional docker compose mechanisms / scripts and any prereqs.